using_images/other_images/jenkins.adoc

```diff
@@ -75,10 +153,50 @@ The Jenkins password can be configured with the following environment variable:
 |Variable name |Description
 
 |`*JENKINS_PASSWORD*`
-|Password for the `*admin*` user.
+|Password for the `*admin*` user when using default Jenkins authentication.
+
+|`*OPENSHIFT_ENABLE_OAUTH*`
+|Determines whether the OpenShift Login plugin manages authentication when logging into Jenkins.  Enabled when set to any non-empty value other than "false".
+
+|`*OPENSHIFT_PERMISSIONS_POLL_INTERVAL*`
+|Specifies in seconds how often the OpenShift Login plugin polls {product-title} for the permissions associated with each user defined in Jenkins.
 
 |===
 
+=== Cross Project Access
+
+If you are going to run Jenkins somewhere other than as a deployment within your same project you will need to provide an access token to Jenkins to access your project.
+
+    1. Identify the service account token secret:
```

Comment:
"Identify the service account token secret associated with a service account that has appropriate permissions to access the project Jenkins will need to access:"